modules/mono/editor/Godot.NET.Sdk/Godot.NET.Sdk/Sdk/Sdk.targets

```diff
@@ -10,9 +10,9 @@
     <!--
     Define constant to determine whether the real_t type in Godot is double precision or not.
     By default this is false, like the official Godot builds. If someone is using a custom
-    Godot build where real_t is double, they can override the GodotRealTIsDouble property.
+    Godot build where real_t is double, they can override the GodotFloat64 property.
     -->
-    <DefineConstants Condition=" '$(GodotRealTIsDouble)' == 'true' ">GODOT_REAL_T_IS_DOUBLE;$(DefineConstants)</DefineConstants>
+    <DefineConstants Condition=" '$(GodotFloat64)' == 'true' ">REAL_T_IS_DOUBLE;$(DefineConstants)</DefineConstants>
```

Comment:
@neikeq Do we want to change the constant name in the Sdk?

Reply:
I couldn't find the old `GODOT_REAL_T_IS_DOUBLE` name in use anywhere

Reply:
This is the Sdk, it defines the constant to be used in game projects.

Reply:
In that case it might be good idea to maintain the GODOT prefix

Reply:
I don't have a strong opinion. The benefit of removing the prefix is consistency. Originally, I used the prefix to make it clear this was a Godot constant and to avoid conflicts, but it's not like there's any code out there using a constant named `REAL_T_IS_DOUBLE` other than Godot.

Reply:
I'll let you guys decide. We can merge once this suggestion is resolved.